pythonturtle/shelltoprocess/console.py

```diff
@@ -1,14 +1,14 @@
 """
 See documentation for the class Console defined here.
 """
-import code
+from code import InteractiveConsole
```

Comment:
I thought we agreed we both like the long module name better?

Reply:
This import fixes a redefinition problem in this module at [lines 74++](https://github.com/bittner/PythonTurtle/blob/master/pythonturtle/shelltoprocess/console.py#L74).

What is this `code` module, by the way? -- Crazy, it's a [standard library module](https://docs.python.org/3/library/code.html). Yikes!